x-pack/solutions/observability/packages/kbn-streams-schema/src/helpers/condition_to_query_dsl.ts

```diff
@@ -62,6 +63,9 @@ export function conditionToQueryDsl(condition: Condition): any {
       },
     };
   }
+  if (isAlwaysCondition(condition)) {
```

Comment:
This is a general fix I stumbled over - right now `always: {}` translates to `match_none: {}`, which is very confusing.